Description: Add missing import to test
 One of the tests was failing do to a missing import.
Author: David Steele <steele@debian.org>


--- a/tests/example_filesystems/example_test_helpers.go
+++ b/tests/example_filesystems/example_test_helpers.go
@@ -7,6 +7,7 @@
 	"testing"
 
 	"github.com/rfjakob/gocryptfs/tests/test_helpers"
+	"github.com/rfjakob/gocryptfs/tests/example_filesystems"
 )
 
 // checkExampleFS - verify that "dir" contains the expected test files
